grails-app/services/org/pih/warehouse/report/ReportService.groovy

```diff
@@ -688,12 +692,36 @@ class ReportService implements ApplicationContextAware {
         return data
     }
 
+    Map getQtyOnOrder (String locationId) {
```

Comment:
Let's spell this out ... so getQuantityOnOrder. Same for variables below and wherever possible.

Reply:
ok